packages/hmr/reactivity/hmr/hooks.py

```
@@ -0,0 +1,44 @@
+from collections.abc import Callable
+from contextlib import contextmanager
+from typing import Any
+
+pre_reload_hooks: dict[str, Callable[[], Any]] = {}
+post_reload_hooks: dict[str, Callable[[], Any]] = {}
+
+
+def pre_reload[T](func: Callable[[], T]) -> Callable[[], T]:
```

Pylint (3.3.7)

[error] Parsing failed: 'expected '(' (reactivity.hmr.hooks, line 9)'

(E0001)